File: SEMESTER 1/C/Cycle 3/sumOfDigits.c
#include <stdio.h>
int main()
{
	int n, temp, m, sum=0;
	printf("Enter a number: ");
	scanf("%d", &n);
	
	temp = n;
	while(n>0)
	{
		m = n % 10;
		sum = sum + m;
		n=n/10;
	}
	printf("Sum: %d\n",sum);
}

File: SEMESTER 2/PHP/sumOfDigits.php
<!DOCTYPE html>
<html>
<head>
    <title>Sum of Digits</title>
</head>
<body>
    <h2>Find Sum of Digits</h2>
    <form method="post" action="<?php echo $_SERVER["PHP_SELF"];?>">
        Enter a number: <input type="text" name="number">
        <input type="submit" name="submit" value="Calculate">
    </form>

    <?php
    if ($_SERVER["REQUEST_METHOD"] == "POST") 
    {
        $number = $_POST['number'];
        $sum = 0;

        for ($i = 0; $i < strlen($number); $i++) 
        {
            $sum += $number[$i];
        }
        echo "<p>Sum of digits of $number is $sum</p>";
    }
    ?>

</body>
</html>
